spec/v2/payload_sanitizer_spec.rb

```diff
@@ -494,6 +499,11 @@ class CustomConverterWithProvidedValue < InputSanitizer::V2::PayloadSanitizer
           sanitizer.should_not be_valid
         end
 
+        it "returns an error when given an collection of raw values" do
+          @params = { :tags => ['nope'] }
```

Comment:
Would be great to also check errors (and json paths, see: https://github.com/zendesk/input_sanitizer/blob/7cf8777d000387f7d567776627cb2804ed5a30e2/spec/v2/payload_sanitizer_spec.rb#L515)

Reply:
Thanks, added :)